### PR TITLE
Query and Update Data

### PR DESCRIPTION
**Pull request by aeh1707**
- [ ] Use database transactions.
- [ ] Modify and delete data in SQL.
- [ ] Prepare complex queries that answer analytical questions.
- [ ] Saving results as a screenshot.

query08
query09
query10
query11
query12
ontent.com/95165255/189493431-5982d55c-0035-4039-919b-fbf7337a2571.PNG)
query14
query15
query16
query17
query18
query19

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE round(weight_kg::numeric, 2) >= 10.23 AND round(weight_kg::numeric, 2) <= 17.3;
+
+BEGIN;
+update animals set species = 'unspecified';
+rollback;
```
Comment (AnselemOdims) [inline queries.sql:14]: - [ ] Please kindly include queries to verify that the change was made before rolling back

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+-- 
+BEGIN;
+update animals set species = 'digmon' where name like '%mon';
+update animals set species = 'pokemon' where species is NULL;
+commit;
```
Comment (AnselemOdims) [inline queries.sql:19]: - [ ] Please kindly include queries to verify the change was made before and after committing

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly add screenshots of the results to the pull request description
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @aeh1707, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

- [ ] descriptive PR ✔️ 
- [ ] Meaningful name of PR ✔️ 

 ## Required Changes ♻️

- [ ] Please as mentioned by the previous reviewer and also as mentioned in the project requirements you should add screenshots of your results to the pull request description 🙃 😉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED 🟢 🟢 

Hi @aeh1707, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ]  Descriptive PR ✔️
- [ ] Meaningful name of PR ✔️
- [ ] Good commit messages ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**